### ISSUE TITLE
Importing Skills with Encumbrance Penalties (Or other penalties).

### ISSUE THREAD

**Issue by ingeanus** [crnormand/gurps#1454]
When you import a skill that takes into account encumbrance or a similar penalty, the Skill Level that gets imported is the skill level that already includes the encumbrance penalty (e.g. if you have **Stealth 12** and are at **Light (1)**, GCS says you have **Stealth 11 with a -1 penalty from encumbrance**; When this is imported into Foundry it is just **Stealth 11**). 

This becomes an issue when those penalties every change. If you switch Encumbrance Levels, lower or offset the penalty, etc. the penalty remains and the onus falls on the user to remember to change the modifier themselves, change the skill level whenever importing, or something else. 
As well, since your equipment that isn't carried (e.g. a gun left in your car) does not import its stats, you cannot just leave equipment off your character to avoid encumbrance penalties and then re-add it after import.

Theoretically as well, this would apply to any bonus or penalty that has the possibility to often change (e.g. a Laser Sight giving +1 to Guns). Some of those seems like they are best left to the user to choose whether they're on by default, but encumbrance in particular seems like one that should be accounted for on import.
Perhaps it is possible to determine modifiers by checking things such as the encumbrance level or skill modifiers and undo certain ones (notably encumbrance) / create on-the-fly modifiers on the skills if it's undone / create a note on the skill / warn the user / etc. Those all have their own serious pros and cons though of course.

**Comment by crnormand** [comment]
Good point.   Unfortunately, there is nothing we can do about it.    At least not in the short term.   We would have to build some of the character creation logic into the GGA to be able to interpret this data... and we would only get the data from GCS (we would have no way of getting it from GCA).

I will keep this issue open, because you never know.   We might slowly get more and more of the creation logic in the GGA, and this might be possible.
